internal/kafka/internal/api/admin/private/api/openapi.yaml

```diff
@@ -349,6 +349,10 @@ components:
           type: string
         href:
           type: string
+      required:
```

Comment:
Btw. I think this is fine but for future this would be breaking change. We need to have tooling for detecting breaking changes to assist API teams.